<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerMatlabModuleGeneratorModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerMatlabModuleGeneratorModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton">
     <property name="text">
      <string>Matlab configuration</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Matlab executable</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ctkPathLineEdit" name="lineEdit_MatlabExecutablePath"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Matlab script directory</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_MatlabScriptDirectory</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_MatlabScriptDirectory">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton">
     <property name="text">
      <string>Matlab module generation</string>
     </property>
     <property name="contentsFrameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Module name</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_GeneratorModuleName</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_GeneratorModuleName"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_GenerateModule">
        <property name="text">
         <string>Generate module</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_GeneratorResults"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QWidget</extends>
   <header>ctkPathLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
